src/components/User/User.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { UserType } from '../../types';
+import './User.css';
+
+export const User = ({ name }) => (
+  <div className="user__name">
+    <b>Name</b>
+    {': '}
+    {`${name}`}
+  </div>
+);
```

Comment:
It's not very rational to create a separate component only just for displaying one string